unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/SettingsPanelHUD/Tests/PlayMode/SettingsWidgetTests_PlayMode.cs

```diff
@@ -0,0 +1,75 @@
+using DCL.SettingsPanelHUD.Controls;
+using DCL.SettingsPanelHUD.Widgets;
+using NSubstitute;
+using NUnit.Framework;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+namespace Tests
+{
+    public class SettingsWidgetTests_PlayMode : TestsBase
```

Comment:
Remember that if the tests don't rely on `SceneController`, `DCLCharacterController`, etc. Inheriting from `TestsBase` is not necessary and we can speed up the test by removing it.

Reply:
You're right! Thanks for the advice :)

Fixed in this new PR (in order to avoid mix things between the part I and II): https://github.com/decentraland/explorer/pull/1784